### ISSUE TITLE
Sandbox pulling takes too much CPU

### ISSUE THREAD

**Issue by M5oul** [duniter/duniter#968]
Certifications and identities pulling take too much CPU.
It would be great to lower this action. It's not necessary to go fast.

**Comment by c-geek** [comment]
Agreed. Today it is every 15 minutes, it is too much and will go worse and worse as the number of newcomers increase.

**Comment by M5oul** [comment]
May be this could happen when the node is excluded of PoW.
Because, when PoW and WoT sandbox pulling are running at the same time, the CPU overbooked.

**Comment by c-geek** [comment]
To make it easier, I'm splitting this issue into 2 parts:

* the too short delay between sync as of today
* the problem of making a pulling during PoW computation

The latter will be for another version than 1.3.0.
